src/main.ts

```diff
@@ -140,6 +141,13 @@ async function main() {
 				if (response) {
 					message.reply(response);
 				}
+
+				const spamPrevention = await messageIsSpam(message);
+				if(spamPrevention){
```

Comment:
Bit a nitpick from me, the style probably could do we being changed to keep it inline with the rest of the code
```suggestion
				if (spamPrevention) {
```